README.md

```diff
@@ -23,7 +23,7 @@ A curated list of awesome things and projects built by Algerian developpers.
   - [Arli](https://github.com/elkebirmed/arli) - JavaScript tools for an Arabic development
   - [react-graphql-redux](https://github.com/youknowriad/react-graphql-redux) - use GraphQL to query your Redux store
   - [Rungen](https://github.com/youknowriad/rungen) - A generic runtime around async flow in javascript
-
+  - [jQuery Ajax Native](https://github.com/acigna/jquery-ajax-native) - jQuery Plugin that adds XHR2 responseType support
```

Comment:
Thanks for this addition @tarikbenmerar if you could just keep the alphabetical order here and move this right after "Arli". Tks a lot